crates/stackable-operator/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Derive `Display` for `LogLevel`([#805]).
```

Comment:
note: We are not actually deriving `Display` (`std::fmt::Display`) directly, but instead `strum::Display`. It of course uses `std::fmt::Display` internally.

I would suggest to change the entry to this:

```suggestion
- Derive `strum::Display` for `LogLevel`([#805]).
```

Reply:
The PR title would also require updating if we move forward with my suggestion.